db/migration/02_seeting_up_peeps_database.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peeps (id SERIAL PRIMARY KEY, user_id INT, posted_time TIMESTAMP, content VARCHAR(280));
```

Comment:
user_id in peeps could be referenced as a foreign key in table creation as it can be linked to the primary key in the user table